### PR TITLE
Specify initrd_imagetype as optional.

### PR DESCRIPTION
**Pull request by drewmoseley**
Signed-off-by: Drew Moseley <drew.moseley@northern.tech>

### CODE REVIEW

**Comment by drewmoseley** [comment]
@kacf can you review this? I can't seem to add reviewers on this repo.

File: 90_mender_boot_grub.cfg [hunk 1]
```diff
@@ -24,7 +24,7 @@ fi
 
 if [ "${drop_to_grub_prompt}" = "no" ]; then
     if linux (${mender_grub_storage_device},${ptable_type}${mender_boot_part})/boot/${kernel_imagetype} root=${mender_root} ${bootargs}; then
-        if test -e (${mender_grub_storage_device},${ptable_type}${mender_boot_part})/boot/${initrd_imagetype}; then
+        if [ test -n "${initrd_imagetype}" -a test -e (${mender_grub_storage_device},${ptable_type}${mender_boot_part})/boot/${initrd_imagetype} ]; then
```
Comment (kacf) [inline 90_mender_boot_grub.cfg]: At least in shell `[ ... ]` is a special construct which is exactly identical to `test ...`. Hence it is incorrect to keep both the square brackets and `test`. I think you can just remove the square brackets and you should be fine.

Comment (drewmoseley) [reply 90_mender_boot_grub.cfg]: I was just mirroring what was already there.  See https://github.com/mendersoftware/grub-mender-grubenv/blob/444738fc26dccc68b1efaf7ab566f0dafff0bed1/90_mender_boot_grub.cfg#L15

I can clean that one up as well if you are certain that it's the right fix.

Comment (drewmoseley) [reply 90_mender_boot_grub.cfg]: Pushed with a second commit to cleanup the other instance.

Comment (kacf) [reply 90_mender_boot_grub.cfg]: Thanks, I guess it must work since it was there already, but definitely not the documented way! Good to have it fixed.

**Pull request merged**